app/src/main/res/layout-v1/theme_entry_card_n.xml

```diff
@@ -7,7 +7,7 @@
     android:foreground="?android:selectableItemBackground"
     card_view:cardBackgroundColor="@color/main_screen_card_background"
     card_view:cardCornerRadius="0dp"
-    card_view:cardElevation="0dp">
+    card_view:cardElevation="2dp">
```

Comment:
rip, I'll revert it
It made the actual theme card more distinguishable from the white subs background